### PR TITLE
Improve formatting in packages and make imports more consistent

### PR DESCRIPTION
**Pull request by invidian**

### CODE REVIEW

File: internal/integration/fetcher.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 	"sync"
 	"time"
 
-	io_prometheus_client "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_model/go"
```
Comment (roobre) [inline internal/integration/fetcher.go:17]: I'm not really a fan of the d-word... Could we use `model` instead?

Comment (invidian) [reply internal/integration/fetcher.go:17]: This is how it is imported in Prometheus code itself: https://pkg.go.dev/github.com/prometheus/common/expfmt. I think we should be following that.

**Comment by invidian** [comment]
Rebased to resolve conflicts.

**Review by roobre — APPROVED** [review]

**Comment by invidian** [comment]
CI blocked on #221.

**Pull request merged**